ui/widgets/resizable.js

```diff
@@ -690,6 +684,21 @@ $.widget( "ui.resizable", $.ui.mouse, {
 		};
 	},
 
+	_calculateAdjustedElementDimensions: function( element ) {
+		if ( !( /content-box/ ).test( element.css( "box-sizing" ) ) ) {
+			return {
+				height: parseFloat( element.css( "height" ) ),
+				width: parseFloat( element.css( "width" ) )
+			};
+		}
+
+		var outerDimensions = this._getPaddingPlusBorderDimensions( element );
+		return {
+			height: element[ 0 ].getBoundingClientRect().height - outerDimensions.height,
+			width: element[ 0 ].getBoundingClientRect().width - outerDimensions.width
```

Comment:
`getBoundingClientRect` takes transforms into consideration while the CSS `width` & `height` - as well as jQuery APIs - do not. Using `getBoundingClientRect` is not valid. We should probably add a test case where the whole content is treated via a scale transform to make sure we didn't break it.